client/src/ui/App.tsx

```diff
@@ -2,6 +2,29 @@ import React, { Component } from 'react';
 import SearchBar from './SearchBar.jsx';
 import "./css/App.css";
 
+var date = new Date();
```

Comment:
Please to move these lines of code inside a function. Consider the render function later in the file.